### ISSUE TITLE
[Feature Request] Support zoom level configuration

### ISSUE THREAD

**Issue by Artemis21** [Flam3rboy/discord-bot-client#35]
"Settings -> appearance -> accessibility -> zoom level" is missing, and it would be great if it wasn't.

**Comment by Artemis21** [comment]
View (top bar) -> Zoom in/out is an adequate replacement
